kuma/core/tests/__init__.py

```diff
@@ -13,6 +13,18 @@
 from ..urlresolvers import split_path
 
 
+def assert_no_cache_header(response):
+    assert 'max-age=0' in response['Cache-Control']
```

Comment:
Nit: I think we should assert 's-maxage' is not in the header as well

Reply:
I added a second commit that does this, thanks @jwhitlock!